src/static/css/index.css

```
@@ -106,41 +107,28 @@ p {
 }
 
 .intro-image-wrapper {
-  -ms-grid-column: 1;
-  -ms-grid-row: 2;
-  grid-area: image;
-  display: none;
-  width: 100%;
-}
-
-.intro-image {
-  background-image: url(/static/images/home-hero.png);
-  background-size: cover;
-  height: 67.5vw;
-  width: 86vw;
-}
-
-.intro-image-year-wrapper {
   -ms-grid-column: 2;
   grid-area: image;
-  width: 100%;
+  max-height: 485px;
+  max-width: 485px;
 }
 
-.intro-image-year {
-  background-image: url(/static/images/home-hero.png);
-  background-size: cover;
-  max-height: 470px;
-  max-width: 458px;
-  height: 41vw;
-  width: 40vw;
+.intro-image-wrapper svg {
+  position: relative;
+  top: 28px;
```

Changing this to `25px` seems to solve the issue @rviscomi raised.

```suggestion
  top: 25px;
```

Also adding this:
```css
.intro-image-wrapper img {
  position: relative;
}
```
puts the image in front so even the slight overlap is less apparent. Not sure if there's a better way?

---

Agree that `25px` is the right positioning value now that I've updated the image canvas.